Fix: greater than equal filter not working

JessChowdhury opened the pull request:
## Description

Closes #3305 

WhereBuilder value for `isGreaterOrEqualTo` was incorrect and causing the filter to not work.

- [X] I have read and understand the [CONTRIBUTING.md](../CONTRIBUTING.md) document in this repository.

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Existing test suite passes locally with my changes

JarrodMFlesch commented:
@JessChowdhury could you add a test for this? You will want to revert the change locally, write the failing test, then add the fix back in. That would be super helpful!

JessChowdhury commented:
> @JessChowdhury could you add a test for this? You will want to revert the change locally, write the failing test, then add the fix back in. That would be super helpful!

Test added @JarrodMFlesch

JarrodMFlesch commented:
@JessChowdhury Looks like a test is failing (not sure if you saw this)

JessChowdhury commented:
> @JessChowdhury Looks like a test is failing (not sure if you saw this)

Fixed - it was failing on the expected number of docs because I hadn't factored in that the previous 2 tests each create a new doc. I've moved this test above the ones that add more docs.
